apps/cli/src/commands/link.ts

```diff
@@ -0,0 +1,57 @@
+import { stdin as input, stdout as output, cwd } from "node:process";
```

Comment:
input, output are quite generic name and often used as arguments. Better pass input:stdin to createInterface for readability.